rust_icu_uloc/src/lib.rs

```diff
@@ -562,6 +674,19 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn test_keywords_nounicode() -> Result<(), Error> {
+        let loc = ULoc::for_language_tag("az-Cyrl-AZ-u-ca-hebrew-t-it-x-whatever")?;
```

Comment:
For completeness, please add key-value pairs to one of the other extensions (e.g. `x-key-whatever-value`).

Reply:
This setting is not implemented at all for the time being.  Once it is, it will make sense to add such a test.